x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/manage_processor_form/processors/kv.tsx

```diff
@@ -33,9 +33,15 @@ const fieldsConfig: FieldsConfig = {
     label: i18n.translate('xpack.ingestPipelines.pipelineEditor.kvForm.fieldSplitFieldLabel', {
       defaultMessage: 'Field split',
     }),
-    helpText: i18n.translate('xpack.ingestPipelines.pipelineEditor.kvForm.fieldSplitHelpText', {
-      defaultMessage: 'Regex pattern for splitting key-value pairs.',
-    }),
+    helpText: () => (
```

Comment:
nit; I'm sure this anonymous function is needed for these helpText entries. Should be able to just have:

```ts
helpText: <FormattedMessage>...</FormattedMessage>
```

Reply:
You're right. Thanks for catching this.